### PR TITLE
[RW-2681][Risk=no] Add back highlight search term concept search

### PR DESCRIPTION
**Pull request by blrubenstein**

### CODE REVIEW

File: ui/src/app/views/concept-table/component.tsx [hunk 1]
```diff
@@ -135,10 +143,35 @@ export class ConceptTable extends React.Component<{concepts: Concept[];
...
+          words.push(splits[i], matches[i]);
+        }
+        words.push(splits[splits.length - 1]);
+      }
+      return words.map(word => <span
+        style={matchString.test(word.toLowerCase()) ? styles.highlighted : {}}>
+          {word}
+        </span>);
+    } else {
+      return stringToHighlight;
```
Comment (NehaBroad) [inline ui/src/app/views/concept-table/component.tsx]: nit: we can also write the above as 
 if (!searchTerm) {
return stringToHighlight;
}
const words....

File: ui/src/app/views/concept-table/component.tsx [hunk 2]
```diff
@@ -27,11 +27,19 @@ const styles = reactStyles({
     marginLeft: 10,
     verticalAlign: 'middle',
     color: colors.blue[3]
+  },
+  highlighted: {
+    backgroundColor: 'yellow',
```
Comment (NehaBroad) [inline ui/src/app/views/concept-table/component.tsx]: if this yellow is same as '#F8C954' we can use colors.ts

File: ui/src/app/views/concept-table/component.tsx [hunk 3]
```diff
@@ -135,10 +143,35 @@ export class ConceptTable extends React.Component<{concepts: Concept[];
...
     </SynonymsObject>);
   }
 
+  highlightWithSearchTerm(stringToHighlight: string) {
+    const {searchTerm} = this.props;
+    if (searchTerm) {
+      const words: string[] = [];
+      let searchWords = searchTerm.split(new RegExp(',| '));
+      searchWords = searchWords.filter(w => w.length > 0 );
+      searchWords = searchWords.map(word => word.replace(/[&!^\/\\#,+()$~%.'":*?<>{}]/g, ''));
```
Comment (NehaBroad) [inline ui/src/app/views/concept-table/component.tsx]: Can we write this as searchTerm.split(new RegExp(',| '))
        .filter(w => w.length > 0 )
        .map(word => word.replace(/[&!^\/\\#,+()$~%.'":*?<>{}]/g, ''))?

File: ui/src/app/views/concept-table/component.tsx [hunk 4]
```diff
@@ -27,12 +27,21 @@ const styles = reactStyles({
     marginLeft: 10,
     verticalAlign: 'middle',
     color: colors.blue[3]
+  },
+  highlighted: {
+    color: colors.green[2],
+    backgroundColor: colors.green[2].alpha(0.2),
+    backgroundOpacity: '0.2',
```
Comment (NehaBroad) [inline ui/src/app/views/concept-table/component.tsx]: question: do we need backgroundOpacity if its specified in backgroundColor?

Comment (blrubenstein) [reply ui/src/app/views/concept-table/component.tsx]: we don't, that's a meaningless css property that doesn't exist that I was trying out and need to remove!

**Pull request merged**